src/storage/mvcc/reader/point_getter.rs

```diff
@@ -0,0 +1,793 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use kvproto::kvrpcpb::IsolationLevel;
+
+use storage::mvcc::write::{Write, WriteType};
+use storage::mvcc::{default_not_found_error, Lock, Result};
+use storage::{Cursor, CursorBuilder, Key, Snapshot, Statistics, Value, CF_LOCK};
+use storage::{CF_DEFAULT, CF_WRITE};
+
+use super::util::CheckLockResult;
+
+/// `PointGetter` factory.
+pub struct PointGetterBuilder<S: Snapshot> {
+    snapshot: S,
+    multi: bool,
+    fill_cache: bool,
+    omit_value: bool,
```

Comment:
Is there a special reason not to use the old name? And it seems we don't support `key_only` in point get.

Reply:
> Is there a special reason not to use the old name

It's simply a more understandable name.

> it seems we don't support key_only in point get.

Yes and it should be a bug.. I remember that there is even a TODO or FIXME in the coprocessor scanner that talking about this missing feature..